IMS.DL/DDL/productSchema.sql

```diff
@@ -0,0 +1,17 @@
+USE IMS;
+GO
+
+IF NOT EXISTS (
+		SELECT 1
+		FROM INFORMATION_SCHEMA.TABLES
+		WHERE TABLE_SCHEMA = 'dbo'
+			AND TABLE_NAME = 'product'
+		)
+BEGIN
+	CREATE TABLE product (
+		id INT PRIMARY KEY IDENTITY(1, 1),
+		name NVARCHAR(50) NOT NULL,
+		price DECIMAL(9, 2) NOT NULL,
```

Comment:
in general when working with money we do gives them more than just 2 decimal, 2 decimal is the realistic number to show for any user but for the sake of calculation and being precise  we save any money value with at lest 4 decimal (its really depending on the requirement).

so in the database and the calculation we use at lest 4 decimals then send pack to the user 2 decimal when show these numbers (here we need to define a money rounding policy).